site/public/js/ta-grading-v2.js

```diff
@@ -20,19 +20,29 @@ let panelElements = [
 
 // Tracks the layout of TA grading page
 const taLayoutDet = {
-  isTwoPanelsEnabled: false,
+  numOfPanelsEnabled: 1,
   isFullScreenMode: false,
   isFullLeftColumnMode: false,
   currentOpenPanel: panelElements[0].str,
   currentTwoPanels: {
-    left: null,
-    right: null,
+    leftTop: null,
```

Comment:
Since the values in here are supposed to be strings can you wrap them with quotes so its explicit that these keys are not variables.